pre-commit

```diff
@@ -42,6 +42,11 @@ if [[ -f "$git_root/.git" ]]; then
     fi
 fi
 
+# Only puppet 3.2.1 - 3.8 support "--parser future" option.
+case $(puppet --version) in
+  4*) USE_PUPPET_FUTURE_PARSER="disabled" ;;
```

Comment:
I would add some information about it to documentation, as now you will need to dig in code to find out, that you can enable it somehow.